coconut/control/control.go

```diff
@@ -262,56 +265,62 @@ func GetEnvironments(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.Com
 func CreateEnvironment(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.Command, args []string, o io.Writer) (err error) {
 	configPayload, err := cmd.Flags().GetString("configuration")
```

Comment:
Initially I thought this was minor and went ahead but I had to go back up to address it because it actually confuses me quite a bit when reading the code: you're calling it `configPayload` but it's actually a `configPayloadKey` or `configPayloadPath` or `configPayloadIdentifier` or `configPayloadEntryName` (pick whichever you prefer). When I write "payload", I'm thinking "content", "value", not "identifier": we use the latter to then fetch the former.